packages/terra-form-input/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ ChangeLog
 
 Unreleased
 ----------
+### Fixed
+* Faded the placeholder text on IE11 by adding vendor specific style rule.
```

Comment:
nit, can we switch this sentence
```suggestion
* Add vendor style rule for IE11 to fade placeholder text.
```

Reply:
changed change logs for all three components : https://github.com/cerner/terra-core/commit/88ec784c6421f99916bf8abcc9723556f18b0bbb